main/guides/getting-started/index.md

```diff
@@ -277,31 +277,10 @@ Once the plug-in has been installed, open Keplr and select the option to **Impor
 <img alt='Choose the "Use recovery phrase or private key" option' src="./assets/040.png"
   width="500" />
 
-To import your wallet, you'll need to copy your mnemonic phrase into Keplr. You can find this series of 24 words back on your terminal window. Copy from this window into your Keplr wallet, then hit the "Import" button.
-
-Please note that your phrase might not be the same as the one shown in this guide!
-
-Note the mnemonic phrase in the output below:
+To import your wallet, you'll need to copy the following mnemonic phrase into Keplr.
```

Comment:
consider leaving a code comment about where the key comes from https://github.com/Agoric/agoric-3-proposals/tree/main/packages/synthetic-chain/public/upgrade-test-scripts/keys_for_test_only

Reply:
small note that the mnemonic is wrong. I tried this out on dApp offer up and it seems that user1 pertains to a different mnemonic on-chain.